src/main/java/org/fundacionjala/coding/kevincristhian/Srot.java

```diff
@@ -0,0 +1,50 @@
+package org.fundacionjala.coding.kevincristhian;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * sort class.
+ */
+public class Srot {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)